tutorials/iot-device-to-device/README.md

```diff
@@ -0,0 +1,78 @@
+# Device to Device demo
+
+Demonstrates how to relay messages from one device to another device using
+Google Cloud functions.
+
+## Setup
+
+Make sure that you have enabled the [Cloud IoT Core API](https://console.cloud.google.com/apis/library/cloudiot.googleapis.com/)
+from the Google Cloud Console.
+
+## Running
+
+### Create a Google Cloud IoT Core topic
+
+From the [Google Cloud Console](https://console.cloud.google.com/iot) create
+a Google Cloud IoT Core Device Registry. When you create it, either set the
+event notification topic to an existing PubSub topic or create a new topic to
+be used for the demo.
+
+### Upload the Google Cloud Function
+
+[The dev2dev instructions](dev2dev/README.md) explain how to upload the
+Google Cloud function to relay messages from one device to another.
+
+Set it to trigger on the topic configured with your device registry.
+
+```
+    gcloud beta functions deploy relayCloudIot \
+        --stage-bucket=gs://your-gcs-bucket \
+        --trigger-topic=your-topic-id
+```
+
+#### Register a Device
+
```

Comment:
I might be helpful to make these a numbered list. Not a huge deal since there are only two items. Your call.